crates/bevy_ui/src/focus.rs

```diff
@@ -56,82 +56,62 @@ pub struct State {
     entities_to_reset: SmallVec<[Entity; 1]>,
 }
 
-/// The system that sets Interaction for all UI elements based on the mouse cursor activity
-#[allow(clippy::type_complexity)]
+pub type NodeQuery<'a> = (
```

Comment:
I don't think this actually makes the code clearer, and it *definitely* shouldn't be `pub`.

Reply:
I mainly introduced it so that we don't need this in 3 different places. But I'm also not really satisfied as it makes it hard to see what is actually in the query. Should I revert it?

Reply:
I think revert it, and then we can see if there are other strategies to avoid reuse.

Reply:
done